src/amber/cli/templates/app/config/application.cr.ecr

```diff
@@ -1,16 +1,116 @@
 require "./initializers/**"
 require "amber"
-
-# load the application_controller before controllers which depend on it
 require "../src/controllers/application_controller"
 require "../src/controllers/**"
 
-# NOTE: Settings should all be in config/environments/env.yml.
-# Anything here will overwrite all environments.
-Amber::Server.configure do |setting|
-  # Server options
-  # setting.name = "<%= display_name %> web application."
-  # setting.port = 80 # Port you wish your app to run
-  # setting.log = ::Logger.new(STDOUT)
-  # setting.log.level = ::Logger::INFO
+# About Application.cr File
+#
+# This is Amber application entry point. This file is responsible for loading
+# initializers, classes, and all application related code in order to have
+# Amber::Server boot up.
+#
+# > We recommend to not modify the order of the require since the order will
+# affect the behavior of the application.
+#
+# With `Amber::Server.configure` block you can redefine the Server configuration
+# settings and use ENVIRONMENT variables and/or values evaluated at runtime.
+#
+#
+Amber::Server.configure do |settings|
```

Comment:
I'm concerned that this will encourage them to configure their environments here instead of using `config/environments/env.yml`. 

Perhaps we should only show a couple examples that relate to ENV vars and show how to optionally override.

Most settings should be set in the env files. Not here!

Reply:
I will add a better note that the YAMLs are first class citizen and only use this if completely necessary.